kernels/mma_m16n16k32_fp32fp8fp8fp32.h

```diff
@@ -0,0 +1,60 @@
+#include <rocwmma/rocwmma.hpp>
+
+#include "precision.h"
+
+using namespace rocwmma;
+
+
+template<>
+class __align__(4) fragment<matrix_a, 16, 16, 32, precision::fp8, row_major> {
+  public:
+    const static size_t num_elements = 2;
+    VRegF32x2 regs;
+};
+
+template<>
+class __align__(4) fragment<matrix_b, 16, 16, 32, precision::fp8, col_major> {
+  public:
+    const static size_t num_elements = 2;
+    VRegF32x2 regs;
+};
+
+template<typename MatrixT, typename DataLayout>
+__device__ void fill_fragment(fragment<MatrixT, 16, 16, 32, precision::fp8, DataLayout>& frag, float value) {
+  for (size_t i = 0; i < frag.num_elements; i++) {
+    frag.regs.data[i] = value;
+  }
+}
+
+// MFMA compiler intrinsic syntax:
+// https://gpuopen.com/learn/amd-lab-notes/amd-lab-notes-matrix-cores-readme/#mfma-compiler-intrinsic-syntax
+inline __device__ void mma_sync_llvm(
+  fragment<accumulator, 16, 16, 32, float>& d,
+  const fragment<matrix_a, 16, 16, 32, precision::fp8, row_major>& a,
+  const fragment<matrix_b, 16, 16, 32, precision::fp8, col_major>& b,
+  const fragment<accumulator, 16, 16, 32, float>& c) {
+
+  const size_t M = 16;
+  const size_t N = 16;
+  const size_t K = 32;
+  const size_t output_elements_per_thread = M * N / __AMDGCN_WAVEFRONT_SIZE__;
+
+  size_t tid_x = threadIdx.x % __AMDGCN_WAVEFRONT_SIZE__;
+  size_t tid_y = threadIdx.x / __AMDGCN_WAVEFRONT_SIZE__;
+
+  size_t mk = tid_y + K * tid_x;
+  size_t kn = tid_x + N * tid_y;
+
+  long amk = a.regs.data[mk];
+  long bkn = b.regs.data[kn];
+  VecT<float, 4> dmn;
+  VecT<float, 4> cmn{0, 0, 0, 0}; // todo: what to put for C here?
```

Comment:
Now that `cmn` is initialized, the comment can be removed:
```suggestion
  VecT<float, 4> cmn{0, 0, 0, 0};
```

Reply:
The remaining issue is that we should somehow obtain this cmn object from the C input matrix, but I'm not sure (yet) how

Reply:
The same holds for `A` and `B`, their corresponding fragments are now also initialized to zero:
```
  fill_fragment(aFrag, static_cast<Tin>(0));         \
  fill_fragment(bFrag, static_cast<Tin>(0));         \
```
For now, I think it's enough to simply remove the comment at `cmn`.